addon/bootstrap.js

```diff
@@ -11,4 +15,9 @@ function shutdown(data, reason) {}
 function install(data, reason) {}
 function uninstall(data, reason) {}
 
-function handleMessage() {}
+function handleMessage(msg, sender, sendReply) {
+  if (msg && msg.funcName === "getTelemetryPref") {
+    let enableTelemetry = prefs.getBoolPref("toolkit.telemetry.enabled");
```

Comment:
You need to `getPrefType` before you `getBoolPref`, otherwise `getBoolPref` can throw if it's unset for any reason. See [MDN](https://developer.mozilla.org/en-US/Add-ons/Code_snippets/Preferences#check_For_Existence) for more details